jenkins/docker_diff.py

```diff
@@ -1,150 +0,0 @@
-#!/usr/bin/env python
```

Comment:
if anyone is still interested in this checkout https://github.com/GoogleCloudPlatform/container-diff for something more mature